packages/e2e-tests/next-app-dynamic-routes/package.json

```diff
@@ -36,7 +36,7 @@
     "aws-sdk": "2.758.0",
     "babel-plugin-istanbul": "6.0.0",
     "babel-plugin-transform-class-properties": "6.24.1",
-    "cypress": "5.1.0",
+    "cypress": "7.2.0",
```

Comment:
Updated when I noticed `ensureRouteNotCached` and similar intercepting commands weren't behaving as expected (they never threw or failed a test before the update).

Reply:
@kirkness hmm yeah I think this was experimental so might be why it wasn't working correctly all the time - hopefully newer versions are more stable. Should also update it in other e2e tests